pkgs/development/python-modules/nixpkgs-plugin-update/nixpkgs-plugin-update/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[project]
+name = "nixpkgs-plugin-update"
```

Comment:
The new name can (and will be) confused with the name of the rewrite of pluginupdate.py (nixpkgs-plugins-updater), let's stick to the old one
https://github.com/NixOS/nixpkgs/pull/380691

Reply:
> Can you suggest one you’d prefer that still contains `nixpkgs` in the name?

`nixpkgs-pluginupdate` is fine. This is the original name, so people already know what `pluginupdate` is (I am talking about Vim in nixpkgs maintainers). Even though it may seem that it can be confused, but missing dash is enough to distinguish them.